build.sbt

```diff
@@ -39,7 +39,7 @@ lazy val frontend = (project in file("modules/frontend"))
      */
     libraryDependencies ++= Seq(
       "org.scala-js" %%% "scalajs-dom" % "2.4.0",
-      "com.raquo" %%% "laminar" % "15.0.0",
+      "com.raquo" %%% "laminar" % "16.0.0",
```

Comment:
thought: had to update it as the samples on the website wrt. `windowEvents` didn't seem to compile otherwise. There was probably an implicit conversion missing or something, I figured I might as well update the lib - seems to work fine.